packages/core/src/features/cohort/availableCohortsSlice.ts

```diff
@@ -1001,7 +1001,7 @@ const slice = createSlice({
           cohortsAdapter.updateOne(state, {
             id: action.meta.arg ?? getCurrentCohort(state),
             changes: {
-              counts: { ...NullCountsData, status: "pending" },
+              counts: { ...NullCountsData, status: "rejected" },
```

Comment:
If there were an error, we would instead set it to rejected as a in the future can either log or visually indicate an error.